packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -166,13 +166,13 @@ export class MirrorNodeClient {
     const mirrorNodeHttpMaxTotalSockets = parseInt(process.env.MIRROR_NODE_HTTP_MAX_TOTAL_SOCKETS || '300');
     const mirrorNodeHttpSocketTimeout = parseInt(process.env.MIRROR_NODE_HTTP_SOCKET_TIMEOUT || '60000');
     const isDevMode = process.env.DEV_MODE && process.env.DEV_MODE === 'true';
-    const mirrorNodeRetries = parseInt(process.env.MIRROR_NODE_RETRIES || '3');
+    const mirrorNodeRetries = parseInt(process.env.MIRROR_NODE_RETRIES || '0'); // we are in the process of deprecation this "feature" (general retries regardless of the flow)
```

Comment:
Nit. "we are in the process of deprecating this feature"

Reply:
✅  Done